### DISCUSSION TITLE
reversearrow hit animation with hidden

### DISCUSSION THREAD

**Discussion by ampersand64** [ppy/osu#23516] [Ideas] [answered]
The reversearrow has a hit animation when playing hidden. In stable, reversearrows disappear when hit. 
This is, believe it or not, a big deal to hidden players. 
It's inconsistent with how the rest of the gameplay elements behave with HD, because it gives too much visual feedback.

**Comment by peppy** [comment]
Can you confirm that you're saying that the animation is *different* when hidden is applied? If so, I can't see anything like this in the code.

If you're implying that it's *always* different then that may be a thing.

**Comment by Joehuu** [comment, marked as answer]
Escalated to https://github.com/ppy/osu/issues/24902.

**Comment by ampersand64** [comment]
Thanks for hearing me out!

Yes, when testing the difference in animations, the reversearrow animations
consistently behaved the way I describe below:

In stable*, *the hidden and nomod hit animations for reverse arrows are
indeed different.
With NM, it grows and fades out slowly, just like the hitcircles.
With hidden, it fades out suddenly and doesn't seem to grow (looks very
similar to the hitcircle numbers' hit animation).

In lazer, the hit animations for reverse arrows are currently identical for
nomod and hidden, which matches their nomod behavior in stable, but not
their hidden behavior.
They appear "hit", just like the regular hitcircles' hit animation. In
other words, they take a few MS to fade out, and grow as they fade.
